src/Pyz/Zed/HelloSpryker/Presentation/Index/index.twig

```diff
@@ -0,0 +1,51 @@
+{% extends '@Gui/Layout/layout.twig' %}
+
+
+{% block section_title %}
+    {{ 'Contact us messages' | trans }}
+{% endblock %}
+
+{% block content %}
+    <table>
+        <thead>
+        <tr>
+            <th>id</th>
+            <th>name</th>
+            <th>message</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for contactUsDataItem in contactUsData %}
+            <tr>
+                <td>{{ contactUsDataItem['IdMessage'] }}</td>
```

Comment:
Twig can work with getters in objects, so you can change it to `contactUsDataItem.idMessage`.